R/execution.r

```diff
@@ -63,6 +63,7 @@ Executor <- setRefClass(
     'Executor',
     fields = list(
         send_response      = 'function',
+        abort_queued_messages = 'function',
```

Comment:
R is weird... You have to pass in the function to make them available in a object? Took a while to find this place...